core/jazz_es-kinesis-log-streamer/components/utils.js

```diff
@@ -136,93 +156,103 @@ function getApiLogsData(payload) {
   return bulkRequestBody;
 }
 
-function getLambdaLogsData(payload) {
-  let bulkRequestBody = '',
+function getLambdaLogsData(payload, callback) {
```

Comment:
Seeing a bunch of spacing changes instead of actual code changes. Can we fix this?

Reply:
Actually this whole file's indentation is messed up. In order to fix one part of this, the whole file needs to be revamped in terms of indentation.

Reply:
I would hate to reformat the indentation as we would lose history of changes. ideally we shouldn't be doing spacing/indentation changes on entire file as much as possible.